src/index.html

```diff
@@ -95,6 +96,56 @@ <h2 class="headline__subtitle">LET’S CONNECT</h2>
         </ul>
       </div>
     </section>
+    <section>
+      <div class="project-modal" id="modal">
+        <div class="project-modal__container">
+          <button class="project-modal__close-button" type="button">
+            <img src="./assets/icons/close-black.svg" alt="Close modal icon" />
+          </button>
```

Comment:
When clicking the `See project` button to check project details, the popup with details about the project appears as expected, but when clicking the close (X) button, the popup is not disappearing. Kindly, fix that. Here is the screen record for that:

https://user-images.githubusercontent.com/34578631/167814999-9f570401-5af3-4697-bcc8-98e4ad79bde2.mov